src/cls/2pc_queue/cls_2pc_queue_types.h

```diff
@@ -38,6 +38,7 @@ struct cls_2pc_reservation
   void dump(ceph::Formatter *f) const {
     f->dump_unsigned("size", size);
     f->dump_stream("timestamp") << timestamp;
+    f->dump_stream("timestamp") << timestamp;
```

Comment:
unintended change? duplicates the line above

Reply:
sorry, Casey. i intended to dump `entries` also. but changed my mind -- thought it should be included in another PR.